File: Fill the Matrix.cpp
class Solution{   
public:
    int minIteration(int N, int M, int x, int y){    
        int ans=0;
		for(int i=1;i<=N;i++)
			for(int j=1;j<=M;j++)
				ans=max(ans,abs(x-i)+abs(y-j));
		return ans;
    }
};

File: Fill the Matrix.py
class Solution:
    def minIteration(self, N, M, x, y):
        #code here
        ans=0
        for i in range(1,N+1):
            for j in range(1,M+1):
                ans=max(ans,abs(x-i)+abs(y-j))
        return ans
